packages/doctor_pet/lib/core/data/doctor.dart

```diff
@@ -0,0 +1,17 @@
+class doctor {
+  String idcustomer;
+  String namedoctor;
+  String email;
+  DateTime day;
+  String phonenum;
+  bool accepted;
```

Comment:
class name must be uppercase 1st character. Property must be uppercase 1st character of each word